editoast/src/schema/errors.rs

```diff
@@ -60,6 +60,31 @@ pub enum InfraErrorType {
     },
 }
 
+impl std::fmt::Display for InfraErrorType {
```

Comment:
You can use `strum_macros::Display` to generate that impl

Reply:
It could be great to add an enum `ErrorType`